src/black/files.py

```diff
@@ -38,7 +38,7 @@ def find_project_root(srcs: Sequence[str]) -> Path:
     project root, the root of the file system is returned.
     """
     if not srcs:
-        return Path("/").resolve()
+        return Path(".").resolve()
```

Comment:
I think this should be `srcs = [Path.cwd()]` to make this work like it does for `-`. In practice, this will make a difference if your pyproject.toml is in a parent directory.

Reply:
That makes sense, updated along with the tests.

Reply:
Unfortunately now tests are failing on Python 3.6 and 3.7. It's not clear what the issue is from an initial lookover. If you look into it that would be great but I wouldn't mind looking into it myself.

Reply:
I managed to replicate and found out it's due to the mock.call interface changing between [3.7](https://docs.python.org/3.7/library/unittest.mock.html#call) and [3.8](https://docs.python.org/3.8/library/unittest.mock.html#call). 3.8+ is backward compatible with the <3.8 format, so it was an easy fix.

This solution feels a bit cryptic, especially if you aren't familiar with the call structure, but it gets the job done. If anyone has better solutions or a different way to test this change, I'd be happy to implement it.